CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 
 * Add `ISetupSequentialResult<TResult>.Returns` method overload that support delegate for deferred results (@snrnats, #594)
 
+#### Changed
+
+* Changed `Mock.Of<T>()` implementation to use much faster simple approach (@informatorius, #547)
```

Comment:
In principle, this is a perfectly fine changelog entry. But may I humbly suggest that you shorten & rephrase it to be even more relevant from the user's perspective? For example:

> * Speed up `Mock.Of<T>()` by approx. one order of magnitude (...)

Reply:
Oh btw., please replace `#547` (issue) with `#598` (this PR).